config/config.go

```diff
@@ -11,3 +13,24 @@ func init() {
 		log.Fatal("Error loading .env file", err)
 	}
 }
+
+// CheckHaveenv checks if all the env variables given as args exist.
+func CheckHaveenv(vars ...string) bool {
```

Comment:
Please fix all references to `os.Getenv` (`databaseService`).

Reply:
Will do :+1: